src/state/routing/utils.ts

```diff
@@ -41,16 +41,21 @@ export function computeRoutes(
         throw new Error('Expected both amountIn and amountOut to be present')
       }
 
+      const routeProtocol: Protocol = getRouteProtocol(route)
```

Comment:
rm `: Protocol`, the typing is unneeded here. It can be inferred from the return value of `getRouteProtocol`.